bundles/org.openhab.binding.androiddebugbridge/README.md

```diff
@@ -0,0 +1,62 @@
DIA_AUDIO_TRACK,KEYCODE_MEDIA_CLOSE,KEYCODE_MEDIA_EJECT,KEYCODE_MEDIA_FAST_FORWARD,KEYCODE_MEDIA_NEXT,KEYCODE_MEDIA_PAUSE,KEYCODE_MEDIA_PLAY,KEYCODE_MEDIA_PLAY_PAUSE,KEYCODE_MEDIA_PREVIOUS,KEYCODE_MEDIA_RECORD,KEYCODE_MEDIA_REWIND,KEYCODE_MEDIA_SKIP_BACKWARD,KEYCODE_MEDIA_SKIP_FORWARD,KEYCODE_MEDIA_STEP_BACKWARD,KEYCODE_MEDIA_STEP_FORWARD,KEYCODE_MEDIA_STOP,KEYCODE_MEDIA_TOP_MENU,KEYCODE_MENU,KEYCODE_META_LEFT,KEYCODE_META_RIGHT,KEYCODE_MINUS,KEYCODE_MOVE_END,KEYCODE_MOVE_HOME,KEYCODE_MUHENKAN,KEYCODE_MUSIC,KEYCODE_MUTE,KEYCODE_N,KEYCODE_NAVIGATE_IN,KEYCODE_NAVIGATE_NEXT,KEYCODE_NAVIGATE_OUT,KEYCODE_NAVIGATE_PREVIOUS,KEYCODE_NOTIFICATION,KEYCODE_NUM,KEYCODE_NUMPAD_0,KEYCODE_NUMPAD_1,KEYCODE_NUMPAD_2,KEYCODE_NUMPAD_3,KEYCODE_NUMPAD_4,KEYCODE_NUMPAD_5,KEYCODE_NUMPAD_6,KEYCODE_NUMPAD_7,KEYCODE_NUMPAD_8,KEYCODE_NUMPAD_9,KEYCODE_NUMPAD_ADD,KEYCODE_NUMPAD_COMMA,KEYCODE_NUMPAD_DIVIDE,KEYCODE_NUMPAD_DOT,KEYCODE_NUMPAD_ENTER,KEYCODE_NUMPAD_EQUALS,KEYCODE_NUMPAD_LEFT_PAREN,KEYCODE_NUMPAD_MULTIPLY,KEYCODE_NUMPAD_RIGHT_PAREN,KEYCODE_NUMPAD_SUBTRACT,KEYCODE_NUM_LOCK,KEYCODE_O,KEYCODE_P,KEYCODE_PAGE_DOWN,KEYCODE_PAGE_UP,KEYCODE_PAIRING,KEYCODE_PASTE,KEYCODE_PERIOD,KEYCODE_PICTSYMBOLS,KEYCODE_PLUS,KEYCODE_POUND,KEYCODE_POWER,KEYCODE_PROFILE_SWITCH,KEYCODE_PROG_BLUE,KEYCODE_PROG_GREEN,KEYCODE_PROG_RED,KEYCODE_PROG_YELLOW,KEYCODE_Q,KEYCODE_R,KEYCODE_REFRESH,KEYCODE_RIGHT_BRACKET,KEYCODE_RO,KEYCODE_S,KEYCODE_SCROLL_LOCK,KEYCODE_SEARCH,KEYCODE_SEMICOLON,KEYCODE_SETTINGS,KEYCODE_SHIFT_LEFT,KEYCODE_SHIFT_RIGHT,KEYCODE_SLASH,KEYCODE_SLEEP,KEYCODE_SOFT_LEFT,KEYCODE_SOFT_RIGHT,KEYCODE_SOFT_SLEEP,KEYCODE_SPACE,KEYCODE_STAR,KEYCODE_STB_INPUT,KEYCODE_STB_POWER,KEYCODE_STEM_1,KEYCODE_STEM_2,KEYCODE_STEM_3,KEYCODE_STEM_PRIMARY,KEYCODE_SWITCH_CHARSET,KEYCODE_SYM,KEYCODE_SYSRQ,KEYCODE_SYSTEM_NAVIGATION_DOWN,KEYCODE_SYSTEM_NAVIGATION_LEFT,KEYCODE_SYSTEM_NAVIGATION_RIGHT,KEYCODE_SYSTEM_NAVIGATION_UP,KEYCODE_T,KEYCODE_TAB,KEYCODE_THUMBS_DOWN,KEYCODE_THUMBS_UP,KEYCODE_TV,KEYCODE_TV_ANTENNA_CABLE,KEYCODE_TV_AUDIO_DESCRIPTION,KEYCODE_TV_AUDIO_DESCRIPTION_MIX_DOWN,KEYCODE_TV_AUDIO_DESCRIPTION_MIX_UP,KEYCODE_TV_CONTENTS_MENU,KEYCODE_TV_DATA_SERVICE,KEYCODE_TV_INPUT,KEYCODE_TV_INPUT_COMPONENT_1,KEYCODE_TV_INPUT_COMPONENT_2,KEYCODE_TV_INPUT_COMPOSITE_1,KEYCODE_TV_INPUT_COMPOSITE_2,KEYCODE_TV_INPUT_HDMI_1,KEYCODE_TV_INPUT_HDMI_2,KEYCODE_TV_INPUT_HDMI_3,KEYCODE_TV_INPUT_HDMI_4,KEYCODE_TV_INPUT_VGA_1,KEYCODE_TV_MEDIA_CONTEXT_MENU,KEYCODE_TV_NETWORK,KEYCODE_TV_NUMBER_ENTRY,KEYCODE_TV_POWER,KEYCODE_TV_RADIO_SERVICE,KEYCODE_TV_SATELLITE,KEYCODE_TV_SATELLITE_BS,KEYCODE_TV_SATELLITE_CS,KEYCODE_TV_SATELLITE_SERVICE,KEYCODE_TV_TELETEXT,KEYCODE_TV_TERRESTRIAL_ANALOG,KEYCODE_TV_TERRESTRIAL_DIGITAL,KEYCODE_TV_TIMER_PROGRAMMING,KEYCODE_TV_ZOOM_MODE,KEYCODE_U,KEYCODE_UNKNOWN,KEYCODE_V,KEYCODE_VOICE_ASSIST,KEYCODE_VOLUME_DOWN,KEYCODE_VOLUME_MUTE,KEYCODE_VOLUME_UP,KEYCODE_W,KEYCODE_WAKEUP,KEYCODE_WINDOW,KEYCODE_X,KEYCODE_Y,KEYCODE_YEN,KEYCODE_Z,KEYCODE_ZENKAKU_HANKAKU,KEYCODE_ZOOM_IN,KEYCODE_ZOOM_OUT  |
```

Comment:
This renders all into one long line in my markdown viewer. Maybe it's better to put these in a list below the table?